templates/linuxbrew.yaml

```diff
@@ -0,0 +1,52 @@
+# Homebrew on Linux.
+#
+# This template does not use macOS as the guest OS, because it seems hard to automate
+# provisioning macOS instances (accept EULA, skip enabling iCloud, set up SSH...).
+#
+# To run Homebrew on macOS with a sandbox user account, Alcoholless can be used instead.
+# The `alclessctl` CLI of Alcoholless is designed to be similar to `limactl`:
+# https://github.com/AkihiroSuda/alcless
+#
+# ```
+# alclessctl create default
+# cd ~/SOME_DIRECTORY
+# alcless brew install xz
+# alcless xz SOME_FILE
+# ```
+#
+# Eventually, Alcoholless may be consolidated into Lima.
+
+minimumLimaVersion: 1.1.0
+
+base:
+- template://_images/ubuntu-24.10
+# Remove this line to disable the mounts
+- template://_default/mounts
+
+# containerd should be installed via Homebrew when Homebrew supports it
+containerd:
+  system: false
+  user: false
+
+provision:
+- mode: system
+  script: |
+    #!/bin/bash
+    set -eux -o pipefail
+    command -v gcc >/dev/null 2>&1 && exit 0
+
+    # nofile has to be increased: https://github.com/Homebrew/brew/issues/9120
+    cat <<'EOF' >/etc/profile.d/99-linuxbrew.sh
+    ulimit -n 65536
+    [ -e /home/linuxbrew/.linuxbrew/bin/brew ] && eval "$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)"
+    EOF
```

Comment:
This could be a `data` script:

```yaml
- mode: data
  path: /etc/profile.d/99-linuxbrew.sh
  contents: |
    ulimit -n 65536
    [ -e /home/linuxbrew/.linuxbrew/bin/brew ] && eval "$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)"
```

Reply:
Thanks, fixed